src/com/google/enterprise/adaptor/ValidatedUri.java

```diff
@@ -60,7 +60,10 @@ public ValidatedUri(String uriString) throws URISyntaxException {
       uri = new URI(uriString);
       url = uri.toURL();
     } catch (MalformedURLException e) {
-      throw new URISyntaxException(uriString, e.getMessage());
+      int index = e.getMessage().indexOf(": ");
```

Comment:
I made it more specific, only spitting the message if the uriString was after the colon.  Otherwise we get the message:
reason: uriString: uriString